frontend/src/bundles/common/components/user-photo-uploader/user-photo-cropper.tsx

```diff
@@ -0,0 +1,122 @@
+import 'cropperjs/dist/cropper.css';
+
+import React, { useCallback,useRef,useState } from 'react';
+import { type ReactCropperElement } from 'react-cropper';
+import { Cropper } from 'react-cropper';
+import { IconContext } from 'react-icons';
+import { BsSave } from 'react-icons/bs';
+import { FaFileUpload } from 'react-icons/fa';
+import { FaArrowRotateRight } from 'react-icons/fa6';
+import { GrZoomIn , GrZoomOut } from 'react-icons/gr';
+
+import styles from './styles.module.scss';
+
+interface UploadCropperProperties {
+    image: string | ArrayBuffer | null;
+    setImage: React.Dispatch<React.SetStateAction<string | ArrayBuffer | null>>
+    setIsModalOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    setCurrentPhoto: React.Dispatch<React.SetStateAction<string>>;
+    handleImageChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
By convention callbacks that pass to component as props should be named with prefix on, rename it to onHandleImage, and also change it fot other props

Reply:
Done. Thank you for explaining the difference